github-actions/oss-readme-template/__tests__/readme.test.ts

```diff
@@ -86,6 +84,7 @@ For more info, visit our website at [https://gomomento.com](https://gomomento.co
   const EXAMPLE_SDK_PROJECT_INFO: SdkProject = {
     type: ProjectType.SDK,
     language: 'WaterLoop',
+    usageExamplePath: path.join(process.cwd(), 'src', 'examples', 'usage.ts'),
```

Comment:
I think everything in `src` gets bundled into the distributable, so you probably want to put this test file into `__tests__`.  You could do `__tests__/examples/usage.ts`.